Neos.Eel/Classes/Helper/StringHelper.php

```diff
@@ -571,6 +571,26 @@ public function length($string)
     }
 
     /**
+     * Return numbers of words used in a string
+     *
+     * @param string $string The input string
+     * @return integer Length of the string
+     */
+    public function wordCount($unicode_string)
+    {
+        // First remove all the punctuation marks & digits
+        $unicode_string = preg_replace('/[[:punct:][:digit:]]/', '', $unicode_string);
+        // Now replace all the whitespaces (tabs, new lines, multiple spaces) by single space
+        $unicode_string = preg_replace('/[[:space:]]/', ' ', $unicode_string);
```

Comment:
can't we combine this replacement with the one above?
Good thing you documented everything nicely, but mostly we try to avoid too many inline comments. 

Maybe you can summarize those comments within the method description?